pkg/tracker/timeOut_test.go

```diff
@@ -4,46 +4,55 @@
 package tracker
 
 import (
-	"context"
-	"math/big"
-	"os"
-	"path/filepath"
 	"testing"
 
-	"github.com/tellor-io/telliot/pkg/common"
+	"github.com/tellor-io/telliot/pkg/config"
 	"github.com/tellor-io/telliot/pkg/db"
-	"github.com/tellor-io/telliot/pkg/rpc"
 	"github.com/tellor-io/telliot/pkg/testutil"
 	"github.com/tellor-io/telliot/pkg/util"
 )
 
 func TestTimeOutString(t *testing.T) {
+	cfg := config.OpenTestConfig(t)
+	DB, cleanup := db.OpenTestDB(t)
+	defer t.Cleanup(cleanup)
 	logSetup := util.SetupLogger()
 	logger := logSetup("debug")
-	tracker := NewTimeOutTracker(logger)
+
+	tracker := NewTimeOutTracker(logger, cfg, DB, nil, nil)
 	res := tracker.String()
 
 	testutil.Assert(t, res == "TimeOutTracker", "should return 'TimeOutTracker' string")
 
 }
 
-func TestTimeOutTracker(t *testing.T) {
+//Can't make the test compile with current setup. Need a contract.Getter connected to the client, but didn't managed to compile this
```

Comment:
That is fine we will refactor how the trackers work so will revisit the test then.